app/components/link-to-username/component.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+const {
+  computed
+} = Ember;
+
+export default Ember.Component.extend({
+  username: null,
+
+  role: computed('username', function() {
+    if (this.get('username').startsWith('@')) {
+      return 'op';
+    } else {
```

Comment:
Shouldn't we have an `else if` for moderators? You can see them e.g. in the #indieweb room Freenode I believe.